everware/git_executor.py

```diff
@@ -0,0 +1,93 @@
+import git
+import re
+from tornado import gen
+from concurrent.futures import ThreadPoolExecutor
+
+
+class GitExecutor:
+
```

Comment:
really? I can't find a rule that says there should be a blank line after `class MyClass:`